service/raft/group0_state_machine.cc

```diff
@@ -178,7 +178,15 @@ future<> group0_state_machine::merge_and_apply(group0_state_machine_merger& merg
         co_return;
     },
     [&] (write_mutations& muts) -> future<> {
-        return write_mutations_to_database(_sp, cmd.creator_addr, std::move(muts.mutations));
+        bool update_service_levels = std::any_of(muts.mutations.begin(), muts.mutations.end(), [] (auto& mut) {
```

Comment:
@kbr- @gleb-cloudius
What is your opinion on this?
I need to reload SL configuration when there are changes to `system.role_members` or `system.role_attributes` tables.
We've been discussing it with @piodul and there is an ongoing discussion about raft changes and updating corresponding subsystems, so we've decided to do it this way and ask for your opinion.
I can create another command type for auth tables, but auth has 4 tables and only 2 of them interest me (so scanning of table ids is unavoidable if we don't want to reload SL configuration on any change to auth).

Reply:
The direction here is the one I want to take for all of the commands instead of having individual c++ types for each one of them.